tests/dracut_tests/test_driver_updates.py

```diff
@@ -433,10 +434,11 @@ class LoadDriversTestCase(unittest.TestCase):
     def test_basic(self, call):
         """load_drivers: runs depmod and modprobes all named modules"""
         modnames = ['mod1', 'mod2']
-        load_drivers({name: [name] for name in modnames})
+        moddict = collections.OrderedDict({name: [name] for name in modnames})
```

Comment:
Actually I don't care about order of items in the dict, if the order does not change. Standard dictionary is created, retyped to OrderedDict and stored into variable `moddict`. This one is then used when dealing with the tested method. Since OrderedDict is a `dict` class child, tested method silently inherits "sorted" behavior when working with the dictionary.